[ libjpeg-turbo:x64-windows ] Cmake Error.

I get the error message :

```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:14 (message):
    Command failed: C:/vcpkg/downloads/cmake-3.8.0-rc1-win32-x86/bin/cmake.exe;--build;.;--config;Release;--target;install;--;/p:VCPkgLocalAppDataDisabled=true;/p:UseIntelMKL=No;/m
    Working Directory: C:/vcpkg/buildtrees/libjpeg-turbo/x64-windows-rel
    See logs for more information:
      C:\vcpkg\buildtrees\libjpeg-turbo\package-x64-windows-rel-out.log
      C:\vcpkg\buildtrees\libjpeg-turbo\package-x64-windows-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:27 (vcpkg_execute_required_process)
  ports/libjpeg-turbo/portfile.cmake:39 (vcpkg_install_cmake)
  scripts/ports.cmake:73 (include)


Error: Building package libjpeg-turbo:x64-windows failed with: BUILD_FAILED
Please ensure sure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: libjpeg-turbo:x64-windows
  Vcpkg version: 0.0.76-56465aabe5e2722d69f58a9af80535697d8c075b
```

Please help!

--- a/ports/libjpeg-turbo/CONTROL
+++ b/ports/libjpeg-turbo/CONTROL
@@ -1,3 +1,3 @@
 Source: libjpeg-turbo
-Version: 1.4.90-1
+Version: 1.5.1-1
 Description: libjpeg-turbo is a JPEG image codec that uses SIMD instructions (MMX, SSE2, NEON, AltiVec) to accelerate baseline JPEG compression and decompression on x86, x86-64, ARM, and PowerPC systems.

--- a/ports/libjpeg-turbo/portfile.cmake
+++ b/ports/libjpeg-turbo/portfile.cmake
@@ -1,10 +1,14 @@
 include(vcpkg_common_functions)
-set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/libjpeg-turbo-1.4.90)
+
+set(LIBJPEGTURBO_VERSION 1.5.1)
+set(LIBJPEGTURBO_HASH "7b89f3c707daa98b0ed19ec417aab5273a1248ce7f98722a671ea80558a8eb0e73b136ce7be7c059f9f42262e682743abcab64e325f82cd4bd1531e0a4035209")
+
+set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/libjpeg-turbo-${LIBJPEGTURBO_VERSION})
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://github.com/libjpeg-turbo/libjpeg-turbo/archive/1.4.90.zip"
-    FILENAME "libjpeg-turbo-1.4.90.zip"
-    SHA512 43c3d26c70a7356bb0832276fe82eead040c3f4aa17df118f91a38615bfacfdfb25fab41965f9ca2b69d18e0b937a1bb753e93fa2c114e01d5174fc1100010b4
+    URLS "https://github.com/libjpeg-turbo/libjpeg-turbo/archive/${LIBJPEGTURBO_VERSION}.zip"
+    FILENAME "libjpeg-turbo-${LIBJPEGTURBO_VERSION}.zip"
+    SHA512 ${LIBJPEGTURBO_HASH}
 )
 vcpkg_extract_source_archive(${ARCHIVE})
 vcpkg_apply_patches(

